routers/web/repo/projects.go

```diff
@@ -513,6 +514,12 @@ func EditProjectBoard(ctx *context.Context) {
 		board.Title = form.Title
 	}
 
+	if len(form.Color) != 0 {
```

Comment:
No need to check, just single assign will do the same

Reply:
Done